public/src/components/ComposableInput.jsx

```diff
@@ -14,7 +14,7 @@ export const InputContainer = styled.div`
   width: 100%;
 `;
 export const StyledInput = styled.input`
-  
+  -webkit-appearance: textfield;
```

Comment:
You shouldn't have to add webkit rules, styled-components pre-processes the css

Reply:
In this case I had to, because the default appearance for webkit on datetime-local fields doesn't behave well with width: 100%, so I had to make it render as a textfield.